callback_plugins/default.py

```diff
@@ -84,8 +84,9 @@ def v2_runner_on_unreachable(self, result):
             self._display.display("fatal: [%s]: UNREACHABLE! => %s" % (result._host.get_name(), self._dump_results(result._result)), color=C.COLOR_UNREACHABLE)
 
     def v2_runner_on_failed(self,result, ignore_errors=False):
-        '''Save last failure'''
+        # Save last failure and set environment variable
```

Comment:
I would expand this comment a bit more to explain how this variable is used by the handlers.